net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bfd_peers.xml

```diff
@@ -0,0 +1,231 @@
+				When the peer is disabled an ‘administrative down’ message is sent to the remote peer.
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
+
+		<field>
+			<name>Source Address/Interface</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Interface</fielddescr>
+			<fieldname>interface</fieldname>
+			<description>Select which interface we should use</description>
+			<type>select_source</type>
+			<source><![CDATA[frr_get_interfaces()]]></source>
+			<source_name>name</source_name>
+			<source_value>value</source_value>
+		</field>
+		<field>
+			<fielddescr>Local Source Address</fielddescr>
+			<fieldname>local_address</fieldname>
+			<description>
+				<![CDATA[
+				Provide a local address that we should bind our peer listener to and the address we should use to send the packets.<br />
+				This option is mandatory for IPv6.
+				]]>
+			</description>
+			<type>select_source</type>
+			<source><![CDATA[frr_get_bfd_source_addresses()]]></source>
+			<source_name>name</source_name>
+			<source_value>value</source_value>
+		</field>
+
+		<field>
+			<name>Advanced Options</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Detect Multiplier</fielddescr>
+			<fieldname>detectmultiplier</fieldname>
+			<description>
+				<![CDATA[
+				Configures the detection multiplier to determine packet loss.
+				<br />
+				The remote transmission interval will be multiplied by this value to determine the connection loss detection timer.
+				<br />
+				The default value is 3.
+				]]>
+			</description>
+			<type>input</type>
+			<size>20</size>
+		</field>
+		<field>
+			<fielddescr>Receive Interval</fielddescr>
+			<fieldname>receiveinterval</fieldname>
+			<description>
+				<![CDATA[
+				Configures the minimum interval that this system is capable of receiving control packets. 
+				<br />
+				The default value is 300 milliseconds.
+				]]>
+			</description>
+			<type>input</type>
+			<size>20</size>
+		</field>
+		<field>
+			<fielddescr>Transmit Interval</fielddescr>
+			<fieldname>transmitinterval</fieldname>
+			<description>The minimum transmission interval (less jitter) that this system wants to use to send BFD control packets.</description>
+			<type>input</type>
+			<size>20</size>
+		</field>
+		<field>
+			<fielddescr>Echo Interval</fielddescr>
+			<fieldname>echointerval</fieldname>
+			<description>Configures the minimal echo receive transmission interval that this system is capable of handling.</description>
+			<type>input</type>
+			<size>20</size>
+		</field>
+		<field>
+			<fielddescr>Echo Mode</fielddescr>
+			<fieldname>echo_mode</fieldname>
+			<description>
+				<![CDATA[
+				Enables or disables the echo transmission mode. This mode is disabled by default.
+				<br />
+				It is recommended that the transmission interval of control packets to be increased after enabling echo-mode to reduce bandwidth usage. For example: transmission-interval 2000.
```

Comment:
Avoid the passive voice if possible. Use phrases like "The best practice is ..."  rather than "It is recommended" (who recommends it?)

Reply:
This is straight from the FRR docs, I've reworded as best I can.